configs/testdata/valkyrie_config.test.main.yml

```diff
@@ -0,0 +1,71 @@
+logging:
```

Comment:
Is it possible to use the already existing example for config? Just concerned about keeping two almost identical config files up to date

Reply:
It was the problem with the ports. The test fails with the default ports

Reply:
Ah right, well guess its a separate issue to tackle if we keep changing these files a lot

Reply:
Maybe i can to the whole field a env parameter, and if it's missing the parsing will use the default value

Reply:
Sure, worth a shot